doc/rose-rug-stem.html

```diff
@@ -459,10 +490,33 @@ <h3 class="alwayshidden">rose ana III</h3>
                   </ul>
                 </li>
               </ul>
+
+              <p>The <code>grepper</code> analysis module also supports some
+              configuration options (specified in the <code>[ana:config]</code> 
+              section of the app or in the <code>[rose-ana]</code> section of 
+              the Rose site/user configuration:</p>
+
+              <ul>
+                <li><b>grepper-report-limit:</b> A numerical value giving the
+                  maximum number of informational output lines to print for 
+                  each comparison.  This is intended for cases where for example
+                  a pattern-matching comparison is expected to match many 
+                  thousands of occurences in the given files; it may not be 
+                  desirable to print the results of every comparison.  After
+                  the given number of lines are printed a special message 
+                  indicating that the rest of the output is truncated will be
+                  produced.</li>
+                <li><b>skip-if-all-files-missing:</b> Can be set to 
+                  <code>.true.</code> or <code>.false.</code>; if active, any
+                  comparison done on files by <code>grepper</code> will be 
+                  skipped if <i>all</i> of those files are non-existent.  In 
+                  this case the task will return as "skipped" rather than 
+                  passed/failed.</li>
+              </ul>
             </div>
 
             <div class="slide">
-              <h3 class="alwayshidden">rose ana IV</h3>
```

Comment:
Slides "rose ana III, IV and V" do not fit into the slide viewer and should be split.